modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/hooks/useDrop.es.js

```diff
@@ -123,6 +123,28 @@ const isDroppingFieldIntoFieldset = (sourceField, targetField) =>
 const isSameField = (targetField, sourceField) =>
 	targetField && targetField.fieldName === sourceField.fieldName;
 
+const isElementsSetOverField = (targetField, sourceField) => {
+	if (targetField) {
+		return (
+			!!Object.keys(targetField).length &&
+			sourceField?.dragType === 'elementSet:add'
+		);
+	}
+
+	return false;
+};
+
+const isElementsSetOverFieldsGroups = (parentField, sourceField) => {
+	if (parentField) {
+		return (
+			!!Object.keys(parentField).length &&
+			sourceField?.dragType === 'elementSet:add'
+		);
+	}
+
+	return false;
+};
+
```

Comment:
Hi @cgmonte. I think we could create just one function since the body is the same just changing the targetField/sourceField.

Reply:
Hi, @alinedoleron. You are absolutely right! I have just pushed an update adding your suggestion. Thanks